lib/utils/utils.js

```diff
@@ -287,6 +288,10 @@ function isStringLiteral(node) {
   return isLiteral(node) && typeof node.value === 'string';
 }
 
+function isAwaitExpression(node) {
+  return node.type === 'AwaitExpression';
+}
```

Comment:
do we need this function? the condition looks simple enough to be inlined to me 🤔